content/en/blog/releases/v0.6.0.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Build v0.6.0"
```

Comment:
I originally also planned to use a more fancy title, but as it is the blog post for a release, it will appear inside the [Releases category](https://shipwright.io/blog/releases/) where I am continuing the naming the way it is for all previous posts.